lib/flutter_local_notifications.dart

```diff
@@ -10,6 +10,20 @@ typedef Future<dynamic> MessageHandler(String message);
 /// The available intervals for periodically showing notifications
 enum RepeatInterval { EveryMinute, Hourly, Daily, Weekly }
 
+class Time {
+  final int hour;
+  final int minute;
+  final int second;
+
+  Time([this.hour = 0, this.minute = 0, this.second = 0]);
```

Comment:
there should be some validation here to restrict the accepted values, possibly through assert statements. Should add documentation as well through comments but you can leave that for me to do